docs/superpowers/plans/2026-05-02-mcp-scientific-paper-v1.md

```
@@ -0,0 +1,1703 @@
+            ? WeightedScorecard.fromFile(weightsPath)
+            : WeightedScorecard.withDefaultWeights();
+    } catch (Exception e) {
+        scorecard = WeightedScorecard.withDefaultWeights();
+    }
+    TriageDecision triage = TriagePolicy.decide(scorecard, extraction);
+    new GraphJsonWriter().write(stem, outputDir, extraction, triage);
+}
+```
+
+Add required imports at top of file:
+```java
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)